ERROR - 2025-06-16 15:43:25 --> Error connecting to the database: mysqli_sql_exception: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\food-1\system\Database\Database.php(72): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\food-1\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\food-1\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\food-1\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#6 C:\xampp\htdocs\food-1\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:create', Array)
#7 C:\xampp\htdocs\food-1\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#8 C:\xampp\htdocs\food-1\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#9 C:\xampp\htdocs\food-1\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\food-1\system\Database\Database.php(72): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\food-1\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\food-1\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\food-1\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#5 C:\xampp\htdocs\food-1\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:create', Array)
#6 C:\xampp\htdocs\food-1\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#7 C:\xampp\htdocs\food-1\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#8 C:\xampp\htdocs\food-1\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#9 {main}
ERROR - 2025-06-16 15:58:50 --> mysqli_sql_exception: Duplicate key name 'email' in C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\food-1\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\food-1\app\Database\Migrations\2025-06-16-154557_CriaTabelaUsuarios.php(97): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\food-1\system\Database\MigrationRunner.php(876): App\Database\Migrations\CriaTabelaUsuarios->up()
#6 C:\xampp\htdocs\food-1\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\food-1\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\food-1\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\food-1\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\food-1\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\food-1\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\food-1\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-06-16 15:59:57 --> mysqli_sql_exception: Duplicate key name 'email' in C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\food-1\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\food-1\app\Database\Migrations\2025-06-16-154557_CriaTabelaUsuarios.php(97): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\food-1\system\Database\MigrationRunner.php(876): App\Database\Migrations\CriaTabelaUsuarios->up()
#6 C:\xampp\htdocs\food-1\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\food-1\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\food-1\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\food-1\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\food-1\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\food-1\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\food-1\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-06-16 23:42:11 --> mysqli_sql_exception: Unknown column 'usuarios.deleted_at' in 'where clause' in C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\food-1\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\food-1\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\food-1\system\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\food-1\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\food-1\system\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
#8 C:\xampp\htdocs\food-1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
#9 C:\xampp\htdocs\food-1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\food-1\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\food-1\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\food-1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\food-1\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-16 23:42:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuarios.deleted_at' in 'where clause'
[Method: GET, Route: admin/usuarios]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\food-1\\public\\index.php')
CRITICAL - 2025-06-16 23:42:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuarios.deleted_at' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\food-1\\public\\index.php')
CRITICAL - 2025-06-16 23:42:11 --> [Caused by] mysqli_sql_exception: Unknown column 'usuarios.deleted_at' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\food-1\\public\\index.php')
ERROR - 2025-06-16 23:46:30 --> mysqli_sql_exception: Unknown column 'usuarios.deleted_at' in 'where clause' in C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\food-1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\food-1\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\food-1\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\food-1\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\food-1\system\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\food-1\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\food-1\system\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
#8 C:\xampp\htdocs\food-1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
#9 C:\xampp\htdocs\food-1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\food-1\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\food-1\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\food-1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\food-1\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-16 23:46:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuarios.deleted_at' in 'where clause'
[Method: GET, Route: admin/usuarios]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\food-1\\public\\index.php')
CRITICAL - 2025-06-16 23:46:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuarios.deleted_at' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\food-1\\public\\index.php')
CRITICAL - 2025-06-16 23:46:30 --> [Caused by] mysqli_sql_exception: Unknown column 'usuarios.deleted_at' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuarios`.`deleted_at` IS NULL', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(679): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Usuarios->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Usuarios))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\food-1\\public\\index.php')
